Source/Painting/SvgColourServer.cs

```diff
@@ -5,15 +5,6 @@ namespace Svg
 {
     public sealed class SvgColourServer : SvgPaintServer
     {
-        /// <summary>
-        /// An unspecified <see cref="SvgPaintServer"/>.
-        /// </summary>
-        public static readonly SvgPaintServer NotSet = new SvgColourServer();
-        /// <summary>
-        /// A <see cref="SvgPaintServer"/> that should inherit from its parent.
-        /// </summary>
-        public static readonly SvgPaintServer Inherit = new SvgColourServer();
-
         public SvgColourServer()

```

Comment:
Is that downwards compatible? These have been public members that have been used in user code.

Reply:
This definition has been moved to base class(`SvgPaintServer`) and backward compatibility is maintained.

Reply:
Ah ok, thanks - I have been confused...